### DISCUSSION TITLE
Request to change tab icon color to be different with standard salesforce tab icon color.

### DISCUSSION THREAD

**Discussion by whaliim-ibs** [tprouvot/Salesforce-Inspector-reloaded#333] [Ideas]
First of all, thank you for developing the Salesforce Inspector Reloaded extension. It's a fantastic tool that significantly enhances my Salesforce productivity.

I have a small suggestion for improving the visual distinction between the Salesforce Inspector Reloaded tab and the standard Salesforce tab. Currently, both tabs use (almost similar) blue color for their icons, which can make it difficult to quickly identify which tab belongs to which application. To enhance usability, it would be helpful if the Salesforce Inspector Reloaded tab icon could be colored differently.
![SF Inspector Reloaded Tabs](https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/126865193/fd5e9136-6711-42ca-b304-06db5a3abec0)

Thank you for your time and attention.

**Comment by tprouvot** [comment]
Hi @whaliim-ibs,

I already had this [suggestions](https://github.com/tprouvot/Salesforce-Inspector-reloaded/issues/197) and I'm considering the options to address this.

When searching for the options, I found this [workaround](https://medium.com/@isaacjoe/changing-color-of-chrome-extension-icon-e69f0ce69453) this is not a long term solution since the extension updates will remove this customization but it is a temporary alternative.

**Reply by whaliim-ibs** [reply]
Hi @tprouvot,

Thanks for the reply and the workaround. I'm glad that this suggestion has been taken into consideration. 
This discussion can be closed.

**Comment by tprouvot** [comment]
Hi @whaliim-ibs,

This functionality will be delivered in 1.23 🚀  and soon on the beta version

**Reply by whaliim-ibs** [reply]
Hi @tprouvot 

Great news! Thanks for the update!
